examples/08_lighting/main.rs

```diff
@@ -0,0 +1,215 @@
+extern crate amethyst;
```

Comment:
I'd really like to either merge the example with the other one, or move the common parts out somewhere to reduce the maintenance burden